ui/src/app/lib/atlasmap-data-mapper/models/transition.model.ts

```diff
@@ -121,9 +121,13 @@ export class FieldActionConfig {
     public populateFieldAction(action: FieldAction): void {
         action.name = this.name;
         action.config = this;
-        action.argumentValues = [];
-        for (const arg of this.arguments) {
-            action.setArgumentValue(arg.name, '0');
```

Comment:
Sorry to hijack, but just a FYI for #229 taker - this line might be a root cause of #229